README.md

```diff
@@ -55,6 +55,12 @@ or you can just install the **latest** release:
 go install github.com/mineiros-io/terramate/cmd/terramate@latest
 ```
 
+#### Using a package manager
+
+- macOS: You can install Terramate on macOS using
```

Comment:
Homebrew is available on Linux too (https://docs.brew.sh/Homebrew-on-Linux) and `terramate`'s Brew formulae works for Linux just fine:
```
> brew install terramate
==> Downloading https://ghcr.io/v2/homebrew/core/terramate/manifests/0.1.1
############################################################################################################# 100.0%
==> Downloading https://ghcr.io/v2/homebrew/core/terramate/blobs/sha256:41830628bb46aff9f4000973f60d406fc5b5fabb1793
==> Downloading from https://pkg-containers.githubusercontent.com/ghcr1/blobs/sha256:41830628bb46aff9f4000973f60d406
############################################################################################################# 100.0%
==> Pouring terramate--0.1.1.x86_64_linux.bottle.tar.gz
✔  /home/linuxbrew/.linuxbrew/Cellar/terramate/0.1.1: 5 files, 8.7MB
==> Running `brew cleanup terramate`...

> which terramate
/home/linuxbrew/.linuxbrew/bin/terramate

> terramate version
0.1.1
```